### PR TITLE
authenticate user, autoformatting, front end prevent multiple saves o…

### PR DESCRIPTION
**Pull request by Conartisttt**
…f same book

### CODE REVIEW

File: client/src/App.css
Comment (Conartisttt) [inline client/src/App.css:1]: prettier autoformat

File: client/src/components/BookList/index.jsx
Comment (Conartisttt) [inline client/src/components/BookList/index.jsx:1]: prettier autoformat

File: client/src/components/Footer/index.jsx
Comment (Conartisttt) [inline client/src/components/Footer/index.jsx:1]: prettier autoformat

File: client/src/components/Header/index.jsx
Comment (Conartisttt) [inline client/src/components/Header/index.jsx:1]: prettier autoformat

File: client/src/index.css
Comment (Conartisttt) [inline client/src/index.css:1]: prettier autoformat

File: client/src/main.jsx
Comment (Conartisttt) [inline client/src/main.jsx:1]: prettier autoformat

File: client/src/pages/Error.jsx
Comment (Conartisttt) [inline client/src/pages/Error.jsx:1]: prettier autoformat

File: client/src/pages/Home.jsx
Comment (Conartisttt) [inline client/src/pages/Home.jsx:1]: prettier autoformat

File: client/src/pages/Library.jsx
Comment (Conartisttt) [inline client/src/pages/Library.jsx:1]: Protect library page by checking if user has a profile. If not, redirect to homepage.
+ prettier autoformat

File: client/src/pages/Login.jsx
Comment (Conartisttt) [inline client/src/pages/Login.jsx:1]: prettier autoformat

File: client/src/pages/Search.jsx
Comment (Conartisttt) [inline client/src/pages/Search.jsx:1]: remove / add comments
if book is already saved in user, do not allow save button to make call to back end (prevent duplicates x3)
+prettier autoformat

File: client/src/pages/Signup.jsx
Comment (Conartisttt) [inline client/src/pages/Signup.jsx:1]: prettier autoformat

File: client/src/pages/SingleBook.jsx
Comment (Conartisttt) [inline client/src/pages/SingleBook.jsx:1]: Rename getProfile function to getUser function - as auth has a .getProfile function and it was confusing to distinguish between the two. 
+prettier autoformat

**Review by dhiduchick — APPROVED** [review]
lgtm

**Pull request merged**